### PR TITLE
Reformat project.json

### PR DESCRIPTION
**Pull request by hishamco**
Nothing just format **project.json** files, arrange elements and remove extra lines

### CODE REVIEW

**Comment by Eilon** [comment]
Not sure how valuable this is on its own...

**Comment by hishamco** [comment]
As I mentioned before just nothing but format :smile: to be identical like the other repos nothing more

File: test/Microsoft.Framework.Localization.Test/project.json [hunk 1]
```diff
@@ -16,5 +11,8 @@
         "System.Reflection": "4.0.10-beta-*"
       }
     }
+  },
+    "commands": {
```
Comment (Eilon) [inline test/Microsoft.Framework.Localization.Test/project.json]: Formatting here is missed up. BTW this is why these changes aren't usually worth it on their own - the overhead of tweaking things often ends up being a greater cost than the returned value.

Comment (hishamco) [inline test/Microsoft.Framework.Localization.Test/project.json]: Sorry for a typo

**Comment by Eilon** [comment]
@kirthik please merge.

**Comment by kirthik** [comment]
merged 01d0b5db22d6ba2d71b025bd08be099529efc20a
